### PR TITLE
49 batch basic attributes

### LINKED ISSUE #49: Realize batch estimation basic attributes
SDK bindings has a batch basic attributes estimation feature. Need implement it.

### PR DESCRIPTION
**Pull request by matemax**
close #49

### CODE REVIEW

File: lunavl/sdk/estimators/face_estimators/basic_attributes.py [hunk 1]
```diff
@@ -215,10 +215,10 @@ def __init__(self, coreEstimator: IAttributeEstimatorPtr):
     #  pylint: disable=W0221
     @CoreExceptionWrap(LunaVLError.EstimationBasicAttributeError)
     def estimate(
-        self, warp: Union[Warp, WarpedImage], estimateAge: bool, estimateGender: bool, estimateEthnicity: bool
+            self, warp: Union[Warp, WarpedImage], estimateAge: bool, estimateGender: bool, estimateEthnicity: bool
```
Comment (Zeskbest) [inline lunavl/sdk/estimators/face_estimators/basic_attributes.py]: no.

File: lunavl/sdk/estimators/face_estimators/basic_attributes.py [hunk 2]
```diff
@@ -243,3 +243,44 @@ def estimate(
...
         return BasicAttributes(baseAttributes)
+
+    @CoreExceptionWrap(LunaVLError.BatchEstimationBasicAttributeError)
+    def estimateBasicAttributesBatch(
+            self, warps: List[Union[Warp, WarpedImage]],
+            estimateAge: bool, estimateGender: bool, estimateEthnicity: bool,
+            aggregate: bool = False,
+    ) -> Tuple[List[BasicAttributes], Union[None, BasicAttributes]]:
+        """
+        Batch estimation a basic attributes from warped images.
```
Comment (Zeskbest) [inline lunavl/sdk/estimators/face_estimators/basic_attributes.py]: Batch basic attributes estimation on warped images.

**Review by Zeskbest — CHANGES_REQUESTED** [review]

File: lunavl/sdk/estimators/face_estimators/basic_attributes.py (hunk 1 above)
Comment (matemax) [reply lunavl/sdk/estimators/face_estimators/basic_attributes.py]: pycharm.
I did a new format these strings

File: lunavl/sdk/estimators/face_estimators/basic_attributes.py (hunk 2 above)
Comment (matemax) [reply lunavl/sdk/estimators/face_estimators/basic_attributes.py]: +

**Pull request merged**